lib/util.js

```diff
@@ -197,6 +198,25 @@ function pad(n) {
   return StringPrototypePadStart(n.toString(), 2, '0');
 }
 
+/**
+ * @param {string} format
+ * @param {string} text
+ * @returns {string}
+ */
+function formatText(format, text) {
+  validateString(format, 'format');
+  validateString(text, 'text');
+  const formatCodes = inspect.colors[format];
+  if (formatCodes === undefined) {
+    throw new ERR_INVALID_ARG_VALUE(
+      'format',
+      `${format}`,
+      `must be one of ${ObjectKeys(inspect.colors).join(',')}`,
+    );
```

Comment:
nit: you might want to call in the validator function so the generated error message stays consistent.
```suggestion
    validateOneOf(format, 'format', ObjectKeys(inspect.colors));
```